source/partials/_related-post.html.slim

```diff
@@ -7,7 +7,7 @@ li
       - post.authors.each do |author|
         => author.name
       ' on
-      = post.createdOn.strftime('%-d %B %Y')
+      = post.created_on.strftime('%-d %B %Y')
```

Comment:
How come `snake_case` is supported now? Is that due to the upgraded `contentful` gem?

Reply:
Yes, due to the update of the contentul gem, see [Field IDs will now be snake_cased following the 2.0.1 Ruby CDA SDK Upgrade](https://github.com/contentful/contentful_middleman/blob/master/CHANGELOG.md#200)